sdk/core/core-client/review/core-client.api.md

```diff
@@ -242,11 +243,7 @@ export interface OperationRequestOptions {
 }
 
 // @public
-export interface OperationResponse {
-    // (undocumented)
-    [key: string]: any;
-    _response: FullOperationResponse;
-}
+export type OperationResponse = unknown;
```

Comment:
Nice! I think `unknown` should always be cleanly castable, so this is better than before!